dev/src/bulk-writer.ts

```diff
@@ -316,10 +316,13 @@ export class BulkWriter {
    * @private
    */
   private _errorFn: (error: BulkWriterError) => boolean = error => {
+    const isRetryableDeleteError =
+      error.operationType === 'delete' &&
+      (error.code as number) === Status.INTERNAL;
     const retryCodes = getRetryCodes('batchWrite');
     return (
       error.code !== undefined &&
```

Comment:
Do we actually need this undefined check? Would retryCodes.include() just return false?

Reply:
nope! removed.